openedx/core/djangoapps/content_tagging/migrations/0009_enable_tagging_as_default.py

```diff
@@ -0,0 +1,22 @@
+from django.db import migrations
+
+from cms.djangoapps.contentstore.toggles import ENABLE_TAGGING_TAXONOMY_LIST_PAGE
+from openedx.core.djangoapps.content_tagging.toggles import CONTENT_TAGGING_AUTO
+
+def create_flags(apps, schema_editor):
```

Comment:
nit: add more more blank line/space between the imports and function def. Not sure why the linter in the CI didn't catch it.

Reply:
Thank you for noticing this! Fixed here: 720ce0b30fadc2202a43030ee3be279e0d090204